presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -609,7 +610,7 @@ private void createHiveTable(ConnectorSession session, ConnectorTableMetadata ta
         }
         List<HiveColumnHandle> columnHandles = getColumnHandles(tableMetadata, ImmutableSet.copyOf(partitionedBy), typeTranslator);
         HiveStorageFormat hiveStorageFormat = getHiveStorageFormat(tableMetadata.getProperties());
-        Map<String, String> tableProperties = getEmptyTableProperties(tableMetadata, !partitionedBy.isEmpty());
+        Map<String, String> tableProperties = getEmptyTableProperties(tableMetadata);
```

Comment:
> What happens if two transactions try to add a new partition to an existing table? This should fail since there is no place where we could mediate the final result, right?

There is. Read `SemiTransactionalHiveMetastore#finishInsertIntoExistingTable`

Reply:
> "Open source hive does not maintain table level statistics for partitioned table", so the table-level stats are going to be off-sync when e.g. Hive is used to create some partitions. Or ?

It is highly unlikely that you would create a table from `Presto` (that's the only way those statistics could be maintained by Presto on INSERT than) and than insert into that table from Hive. Technically we can add a feature flag, but i don't think it is worth additional code for testing.